kpm/spec/kpm/unit/account_spec.rb

```diff
@@ -0,0 +1,347 @@
+require 'spec_helper'
+
+describe KPM::Account do
+
+  shared_context 'account' do
+    let(:logger) {logger       = ::Logger.new(STDOUT)
+    logger.level = Logger::FATAL
+    logger}
+    let(:dummy_data_file) {Dir.mktmpdir('dummy') + File::SEPARATOR + 'kbdump'}
+    let(:url) {'http://127.0.0.1:8080'}
+    let(:killbill_api_key) {'bob'}
+    let(:killbill_api_secrets) {'lazar'}
+    let(:killbill_user) {'admin'}
+    let(:killbill_password) {'password'}
+    let(:db_name) {'killbill'}
+    let(:db_username) {'root'}
```

Comment:
Quite risky as a default! 😸 

Could you instead use a specific user and database (e.g. 'test_kpm') and update the `README` to notify developers about the setup required to run these tests (database, credentials, etc.)?